models/organization.go

```diff
@@ -0,0 +1,218 @@
+package models
+
+import (
+	"errors"
+	"time"
+
+	"github.com/jinzhu/gorm"
+	"github.com/lib/pq"
+)
+
+// Organization represents a company using white-labeled Temporal
+type Organization struct {
+	gorm.Model
+	// the name of the organization
+	Name string `gorm:"type:varchar(255);unique"`
+	// the corresponding temporal user account that manages this org
+	UserOwner string `gorm:"type:varchar(255);unique"`
+	// the usd value owed by the organization
+	AccountBalance float64 `gorm:"type:float"`
+	// the user accounts who have signed up under this organization
+	RegisteredUsers pq.StringArray `gorm:"type:text[];column:registered_users"`
+}
+
+// OrgManager is an organization model manager
+type OrgManager struct {
+	DB *gorm.DB
+}
+
+// NewOrgManager instantiates an OrgManager
+func NewOrgManager(db *gorm.DB) *OrgManager {
+	return &OrgManager{DB: db}
+}
+
+// NewOrganization is used to create a new organization
+func (om *OrgManager) NewOrganization(name, owner string) error {
```

Comment:
New... usually returns an object. May be set organization is a better name

Reply:
i set it to return an org object